generator.json

```
@@ -1860,6 +1860,93 @@
                     "$include.commonTypeMap": "build/csharp_typemap.json"
                 }
             ]
+        },
+        {
+            "profileName": "webgpu",
+            "sources": [
+                "build/submodules/webgpu-headers/webgpu.h"
```

Given this is just one header file, do we need a submodule?
```suggestion
                "https://raw.githubusercontent.com/webgpu-native/webgpu-headers/main/webgpu.h"
```

---

I dont think a submodule is *nesessary* but given the headers are still being actively worked on, it may be best to keep it as a submodule incase they do any breaking changes, instead of directly referencing the main branch

---

ok fair